protocol/contracts/admin/VestingPayments.sol

```diff
@@ -143,12 +116,14 @@ contract VestingPayments is IVestingPayments, PaymentBase, TempleElevatedAccess
         VestingSchedule storage _schedule = schedules[_vestingId];
         if (_schedule.start == 0) { revert CommonEventsAndErrors.InvalidParam(); }
         if (_schedule.recipient != msg.sender) { revert CommonEventsAndErrors.InvalidAccess(); }
-        if (_schedule.revoked && revokedAccountsReleasable[msg.sender] > 0) {
+        if (_schedule.revoked) {
             uint256 amount = revokedAccountsReleasable[msg.sender];
-            revokedAccountsReleasable[msg.sender] = 0;
-            _release(_schedule, amount);
-            emit Released(_vestingId, msg.sender, amount);
-            return;
+            if (amount > 0) {
```

Comment:
nit: just curious to know if this has gas benefits, that is removing the `revokedAccountsReleasable[msg.sender] > 0` check from if condition and bringing it down to line 121.